identifiers/country-jp/single_member_constituencies.csv

```diff
@@ -0,0 +1,290 @@
+id,name
+ocd-division/country:jp/prefecture:jp-01/ward:1,Constituency 1

```

Comment:
For the names of these districts consider adding the prefecture name as a part of it so that it's clearer what they are.